README.md

```diff
@@ -186,6 +186,113 @@ Now, during the sysroot creation process, you should see the contents of `someth
 NOTE: For trivial text files, as in the example above, you could just as easily create those files fully within the `--custom-setup-script`. However, for binary data such as precompiled libraries, this feature comes to the rescue.
 
 
+## Tutorial
+
+For a new user, this section walks you through a representative use case, step by step.
+
+In this tutorial, we will cross-compile the File Talker tool https://github.com/ros-tooling/file_talker against ROS2 Dashing, to run on an ARM64 Ubuntu system.
+This workflow can be generalized to any `.repos` file for your project.
+
+NOTE: This tutorial currently assumes a Debian-based (including Ubuntu) Linux distribution as the host platform.
+
+### Creating a cross-compilation workspace
+
+1. Create a directory for your workspace
+    * `mkdir cross_compile_ws`
+    * `cd cross_compile_ws`
+1. Create a `.repos` file for `vcs`
+    * `file_talker.repos`
+    ```
+    repositories:
+      file_talker:
+        type: git
+        url: https://github.com/ros-tooling/file_talker.git
+        version: master
+    ```
+1. Set up the sysroot environment for the cross-compiler to use
+    * `mkdir -p sysroot/qemu-user-static/`
+    * `mkdir -p sysroot/ros_ws/src/`
+    * `cp /usr/bin/qemu-*-static sysroot/qemu-user-static`
+    * `vcs import ros_ws/src < file_talker.repos`
+
+### Running the Cross-Compilation
```

Comment:
30:5    warning  'Running the                    Google.Headings     
                  Cross-Compilation' should use                       
                  sentence-style capitalization.